### ISSUE TITLE
Additional +/- buttons

### ISSUE THREAD

**Issue by stefanct** [opening-hours/opening_hours.js#335]
There exist a number of quick-access buttons to change the "current" date and time. As of now there are buttons to jump forward/backward one year, day, hour, and minute respectively.
I was missing buttons for +/- one week. This makes a lot of sense since the graphical representation is also one week long.
Additionally I think +/- one month would also be helpful sometimes.

If you think that costs too much space... the hour and minute (and year) buttons exist already in the form of arrow buttons of the number inputs (in most browsers at least).

**Comment by ypid** [comment]
I am fine with that. Pretty simple to implement. Except that we need to make it translatable and translate it into a few languages. Want to give it a try?

**Comment by stefanct** [comment]
I have looked at the code and it looks (to me!) trivial for "week" (1 line for functionality, +1 line for each language), but month is less straight forward if we truly want it to jump the length of the current/previous month instead of a fixed amount of time (cf. https://github.com/opening-hours/opening_hours.js/blob/master/index.html#L70). Since it's a low-hanging fruit I would simply add support for "week" and let the issue open for "month".

Unfortunately, finding out how I could test this is particular feature is way harder than adding the code :) Just opening the .html does not work. `make check-all` which might be complete overkill requires some node.js binaries already and I really don't want to go that way... I can post my untested patch if you like but it's really trivial.

**Comment by ypid** [comment]
Right, jumping by month is not clearly defined. There is always something complex even in such simple looking cases. Feel free to open the unfinished PR. I will test it. Also feel free to add yourself as contributor to the https://github.com/opening-hours/opening_hours.js/blob/master/CHANGELOG.rst

**Comment by stefanct** [comment]
#336

**Comment by ypid** [comment]
Tested, merged and deployed #336, thanks. As I looked at the code, this all needs to be rewritten from scratch, so I will leave this issue here open until then.
